package.json

```diff
@@ -68,15 +70,17 @@
 		"axios": "^1.6.7",
 		"bcryptjs": "^2.4.3",
 		"blob-stream": "^0.1.3",
+		"dotenv-cli": "^7.4.1",
 		"flowbite": "^2.2.1",
 		"flowbite-svelte": "^0.44.21",
 		"flowbite-svelte-blocks": "^0.5.1",
 		"flowbite-svelte-icons": "^0.4.5",
 		"minio": "^7.1.3",
 		"pdfmake": "^0.2.9",
+		"query-string": "^9.0.0",
 		"svelte-xstate": "^1.0.1",
 		"sveltekit-i18n": "^2.4.2",
-		"sveltekit-superforms": "^1.13.1",
+		"sveltekit-superforms": "^2.9.0",
```

Comment:
It is good practice to always talk to the codeowner before making a change to dependencies of this type

Reply:
My apologies for that, it was vital to use the updated features of this module, I'll take ur advice from now on